Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -4923,7 +4923,7 @@ - (void)reloadViewItems
                             || viewItem.shouldShowDate);
                 }
                 if (shouldShowSenderName) {
-                    senderName = [self.contactsManager displayNameForPhoneIdentifier:incomingSenderId];
+                    senderName = [self.contactsManager contactOrProfileNameForPhoneIdentifier:incomingSenderId];
```

Comment:
Is this styled appropriately? e.g. we generally want to use the attributed flavor to make sure profile names are distinctly recognizable from something fetched from our own contacts.

Reply:
I was wondering the same thing. I'll add a note to discuss with myles.

Reply:
Talked with Myles.  Design isn't final, but I've made some changes per him.